src/App.tsx

```diff
@@ -15,36 +15,127 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
+type ReorderOptions = {
+  sortType: SortType;
+  isReversed: boolean;
+};
+
+// Use this function in the render method to prepare goods
```

Comment:
why do you need this comments?

Reply:
it is from previous task :) will delete it